static/app/components/events/aiAutofix/useAiAutofix.tsx

```diff
@@ -0,0 +1,70 @@
+import {useCallback, useEffect, useRef, useState} from 'react';
+
+import {EventMetadataWithAutofix} from 'sentry/components/events/aiAutofix/types';
+import {Group} from 'sentry/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+const POLL_INTERVAL = 5000;
+
+export const useAiAutofix = (_group: Group) => {
+  const organization = useOrganization();
+  const api = useApi();
+  const {
+    data: group,
+    isError,
+    refetch: dataRefetch,
+    error,
+    dataUpdatedAt,
+  } = useApiQuery<Group>([`/organizations/${organization.slug}/issues/${_group.id}/`], {
+    staleTime: Infinity,
+    retry: false,
+  });
```

Comment:
We also want to disable this query when the autofix is already available with the `enabled` option